roles/shiftstack/tasks/deploy_shiftstackclient_pod.yml

```diff
@@ -43,3 +43,11 @@
       type: Ready
       status: "True"
     proxy: "{{ cifmw_shiftstack_proxy | default(omit) }}"
+
+- name: "Copy the underlying OCP kubeconfig to '{{ cifmw_shiftstack_client_pod_name }}'"
```

Comment:
Rather than adding this task in the deploy_shiftstackclient_pod.yml file I think the main.yml is a better location as it's not part of the pod deployment.